diff --git a/Kotlin-compose/build/compose-benchmarks-benchmarks-wasm-js.uninstantiated.mjs b/Kotlin-compose/build/compose-benchmarks-benchmarks-wasm-js.uninstantiated.mjs
index 444dd97..0424c91 100644
--- a/Kotlin-compose/build/compose-benchmarks-benchmarks-wasm-js.uninstantiated.mjs
+++ b/Kotlin-compose/build/compose-benchmarks-benchmarks-wasm-js.uninstantiated.mjs
@@ -108,8 +108,8 @@ export async function instantiate(imports={}, runInitializer=true) {
              },
         'kotlin.js.__convertKotlinClosureToJsClosure_(()->Unit)' : (f) => getCachedJsObject(f, () => wasmExports['__callFunction_(()->Unit)'](f, )),
         'kotlin.js.jsThrow' : (e) => { throw e; },
-        'kotlin.io.printError' : (error) => console.error(error),
-        'kotlin.io.printlnImpl' : (message) => console.log(message),
+        'kotlin.io.printError' : (error) => printErr(error),
+        'kotlin.io.printlnImpl' : (message) => print(message),
         'kotlin.js.jsArrayGet' : (array, index) => array[index],
         'kotlin.js.length_$external_prop_getter' : (_this) => _this.length,
         'kotlin.js.stackPlaceHolder_js_code' : () => (''),
